sorts/bogo_sort.ts

```diff
@@ -1,11 +1,14 @@
+import { isSortedArray } from "../other/is_sorted_array";
+import { shuffleArray } from "../other/shuffle_array";
+
 /**
  * @function bubbleSort
  * @description bogo sort is very simple to understand, it randomly shuffeles the input array until it is sorted
  * @Complexity_Analysis
  * Space complexity - O(1)
  * Time complexity
- *      Best case   -   O(1)
- *                      The best case occurs when the first shuufle sorts the array.
+ *      Best case   -   O(n)
+ *                      The best case occurs when the array is already sorted.
  *      Worst case  -   unbounded
  *                      The worst case occurs when the suffeles never make the array sorted.
  *      Average case -  unbounded
```

Comment:
You haven't fixed the analysis of the average case.

Reply:
please take a look at the last commit